### DISCUSSION TITLE
[Script request] Silverbullet

### DISCUSSION THREAD

**Discussion by piliener** [community-scripts/ProxmoxVE#127] [Request script]
### Application Name

SilverBullet

### Website

https://silverbullet.md/

### Description

SilverBullet is a note-taking application optimized for people with a _hacker mindset_. SilverBullet is open source **personal knowledge management system** implemented as an offline-capable web application. 

I think this is an ideal fit with the philosophy of this community and is designed to be self hosted. The install process has both options for Docker and Directly with Deno if we wanted to offer the option for both. There is no inherent systemd service built in yet so we'll have to put that into the script itself.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by dsiebel** [comment]
Super intrigued by Silverbullet so I started building the install and update scripts 
while installing it on a clean Debian LXC: https://github.com/community-scripts/ProxmoxVE/pull/659

**Comment by MickLesk** [comment]
Added.
